docs/development/standards.rst

```diff
@@ -1,6 +1,9 @@
 Development Setup and Standards
```

Comment:
If this is the new "install" doc, we should rename this file and move the standards bits to the bottom. Calling this `standards.rst` but having it be the install docs feels weird.

Reply:
I'm fine with that.

Reply:
We should do it as part of this PR, since we're removing the install doc.